server/app/services/question/PrimaryApplicantInfoTag.java

```diff
@@ -0,0 +1,105 @@
+package services.question;
+
+import com.google.common.collect.ImmutableList;
+import models.QuestionTag;
+import services.question.types.QuestionType;
+
+/**
+ * Defines tags that can be applied to a question to denote the answer is a primary piece of
+ * information about the applicant. These are 1:1 with particular QuestionTags.
+ */
+public enum PrimaryApplicantInfoTag {
+  APPLICANT_DOB(
+      QuestionTag.PRIMARY_APPLICANT_DOB,
+      QuestionType.DATE,
+      "primaryApplicantInfoDob",
```

Comment:
question (non-blocking): Do we want to be consistent in whether or not we're using "info" in the name? I'm thinking since the names do not include "info" (eg. `PRIMARY_APPLICANT_DOB`) we might want to leave it out of the field names to (eg. `primaryApplicantDob`). I really don't feel strongly about this, but wanted to raise it since it occurred to me and naming is hard.